localization/i18n/ru/OrcaSlicer_ru.po

```diff
@@ -10963,9 +11118,27 @@ msgid ""
 "and for when tool changing.\n"
 "\n"
 msgstr ""
```

Comment:
@SoftFever I fixed two of the bugs, but there was a problem with the third. I have reconsidered the formation of the string itself and I don't understand what it is complaining about:

> ./localization/i18n/ru/OrcaSlicer_ru.po:11120: 'msgstr' is not a valid C format string, unlike 'msgid'. Reason: The character that terminates the directive number 1 is not a valid conversion specifier.

In this place, the only thing to pick on is the “%” symbol. But it occurs once, just like in the original text.

I also compared its occurrence with other translation keys, as well as the `c-format` format, and I don't understand what could be the reason. Any ideas how to fix it?

Reply:
I've removed the format checking, but I don't know if that's the right thing to do. Although after that the tests passed successfully.

![image](https://github.com/user-attachments/assets/423014f6-f488-4384-a1f8-f3253714fb0f)

Reply:
This change is fine.
Sometimes the get text will confuse.

Reply:
Got it 👍